### ISSUE TITLE
Diff Font Size: not changed by Preferences > General > Font Size

### ISSUE THREAD

**Issue by rsuzuki0** [sublimehq/sublime_merge#1162]
**Version info**

 - OS: 10.15.7
 - Build: 2056

**Description**

Preferences > General > Font Size only changes the font size for the commit comments. It has no effect on the diff display.

**Steps to reproduce**

1. Open a repo, select a file/commit to show some diff.
2. Change the setting in Preferences > General > Font Size
3. Notice the value only changes the font size for the commit comments, not the diff. That remain the same even when you switch around the file or commit.

**Comment by dpjohnst** [comment]
Hi @rsuzuki0,

Thank you for sharing this feedback with us - much appreciated!

To clarify, are you referring to the font size of the file path (shown below)?
![Diff Title Text](https://user-images.githubusercontent.com/18252447/120144679-b7503e80-c225-11eb-98fb-728dd7c1975e.png)

The styling of all basic UI elements are controlled via the theming system. 
If you're looking to increase/decrease the font size shown above, you could try the following:
1. Open Sublime Merge
2. Navigate to `Preferences > Browse Packages`
3. Navigate to the `User` folder
4. Create a file called `Merge.sublime-theme`
5. Add the following to the file:
```
{
    "variables":
    {
        // Global font size
        "font_size": 14,
    },
    "rules":
    {
    },
}
```

I'll be making a note to adjust the preferences to indicate that `font size` only applies to diffs.

Hope this helps!
\- Dylan

**Comment by rsuzuki0** [comment]
No, I'm talking about the actual diff display, not the file path. It 
doesn't change at all in all my computers (two Catalina, one Mojave).

Also, I can't change the font size in Sublime Text 4 with Cmd +/-.

I recall all these worked fine with previous version of both programs.

**Comment by dpjohnst** [comment]
Hi @rsuzuki0,

Thanks for clarifying - perhaps you have a syntax-specific override in place?

To confirm, could you start Sublime Merge in safe mode.
a) If you have the `smerge` command line helper set up, you can run `smerge --safe-mode`
b) Alternatively, close Sublime Merge completely and hold the `option` key as you restart Sublime Merge

A confirmation dialog will appear to confirm you are in safe mode. If the `font_size` option now works, it's likely that you have a syntax-specific font size in place in either Sublime Merge or Sublime Text.

To check for syntax-specific overrides in Sublime Text:
1. Open any of the files you're seeing the issue with in Sublime Text
2. Open `Preferences > Settings (Syntax Specific)
3. Look for any reference to `font_size` within the settings (and remove it)

To check for syntax-specific overrides in Sublime Merge
1. Open Sublime Merge
2. Navigate to `Preferences > Browse Packages`
3. Navigate to the user folder
4. Look for a `.sublime-settings` file matching the syntax you're seeing the issue with
5. Look for any reference to `font_size` within the file (and remove it)

Let me know if this helps!

Thanks,
\- Dylan

**Comment by rsuzuki0** [comment]
Thanks for the info. What you described was the case.

However, I prefer larger fonts on the editor than showing diffs on 
Sublime Merge. I typically use ST4 and SM in different window sizes, 
often on different displays, and I use different levels of focus on 
writing versus checking. Is there a way to decouple the font 
specifications given to the ST4, whereby Sublime Merge ignores or 
overrides them?

Similar challenge happens when I open two files in vertically split 
configuration in ST4. I want the font and wrap settings to adapt to 
the column width. But that's a different topic for now.

**Comment by dpjohnst** [comment]
Hi @rsuzuki0,

Thanks for the additional information!

The order of precedence currently for settings is as below:
* General Settings
* Syntax Specific Settings (Sublime Text)
* Syntax Specific Settings (Sublime Merge)

I would suggest adding the same specific syntax settings override to Sublime Merge as you're overriding in Sublime Text.
For example, if you're overriding `C++` in Sublime Text, then you would do the following in Sublime Merge:
1. Open Sublime Merge
2. Navigate to `Preferences > Browse Packages...`
3. Navigate to the `User` folder
4. Create a file called `C++.sublime-settings` (replace `C++` with the same syntax you're overriding in Sublime Text)
5. Add the following to the file
```
{
	"font_size": 12,
}
```

Thanks,
\- Dylan
